src/components/question-posted-page/question-about.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import './question-posted.scss';
+import Question from './../question/question';
+
+class QuestionAbout extends Component {
+  state = {
+    question: null
+  }
+
+  componentWillMount(){
+    //Sets question state to question matching the pararameters given
+    const token = localStorage.getItem('token');
+    fetch(`${process.env.REACT_APP_END_POINT_URL}/questions/asked`, {
+      method: 'GET', 
+      headers : { 
+        'Authorization' : 'Bearer ' + token,
+        'Content-Type': 'application/json'
+      }})
+    .then(res => res.json())
+    .then(res => 
+      this.setState({
+        question: res.filter(ele => {
+          if (ele.questionId == window.location.pathname.replace(/\D/g, "")) {
```

Comment:
I would add === instead of ==

Reply:
@nataliaero the types are different but i'll add ele.questionId.toString()

Reply:
@nataliaero done!